posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -126,6 +126,32 @@ def where(self):
                 ),
             )
 
+        if self.query.searchQuery:
+            # TODO: Refine this so it only searches the frames inside $exception_list
+            props_to_search = ["$exception_list", "$exception_stack_trace_raw", "$exception_type", "$exception_message"]
+            or_exprs: list[ast.Expr] = []
+            for prop in props_to_search:
+                or_exprs.append(
+                    ast.CompareOperation(
+                        op=ast.CompareOperationOp.Gt,
+                        left=ast.Call(
+                            name="position",
+                            args=[
+                                ast.Call(name="lower", args=[ast.Field(chain=["properties", prop])]),
+                                ast.Call(name="lower", args=[ast.Constant(value=self.query.searchQuery)]),
+                            ],
+                        ),
+                        right=ast.Constant(value=0),
+                    )
+                )
+
+            if or_exprs:
```

Comment:
Why the `if`? Looks like there will always be 4 expressions in `or_exprs`

Reply:
some typing annoyance but I think I fixed that already, will remove 👍